pkg/types/vsphere/validation/platform.go

```diff
@@ -104,3 +110,29 @@ func validateFolder(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+func validateNetwork(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	vim25Client, _, err := vspheretypes.CreateVSphereClients(context.TODO(),
+		p.VCenter,
+		p.Username,
+		p.Password)
+
+	if err != nil {
+		return append(allErrs, field.InternalError(fldPath, errors.Wrapf(err, "unable to connect to vCenter %s.", p.VCenter)))
+	}
+
+	finder := find.NewFinder(vim25Client)
+	ctx := context.TODO()
+
+	dataCenter, err := finder.Datacenter(ctx, fmt.Sprintf("./%s", p.Datacenter))
```

Comment:
Check out the logic being add in https://github.com/openshift/installer/pull/4894/files#diff-cdefcb6fa878811aa273941a4ac54baf6ce971ab287ef81f2736a9d9904cbd61R55-R61 for name of the datacenter.